public/app/features/alerting/unified/components/rule-viewer/v2/RuleViewer.v2.tsx

```diff
@@ -49,124 +88,358 @@ const RuleViewer = ({ match }: RuleViewerProps) => {
+  const runbookUrl = annotations[Annotation.runbookURL];
+  const dashboardUID = annotations[Annotation.dashboardUID];
+  const panelID = annotations[Annotation.panelID];
+
+  const hasPanel = dashboardUID && panelID;
+  const hasDashboardNoPanel = dashboardUID && !panelID;
+
+  return (
+    <>
+      <Stack direction="row" gap={4}>
+        {runbookUrl && (
+          <MetaText direction="column">
+            Runbook
+            <Link href={runbookUrl} size="sm" external>
+              {/* TODO instead of truncating the string, we should use flex and text overflow properly to allow it to take up all of the horizontal space available */}
+              {truncate(runbookUrl, { length: 42 })}
+            </Link>
+          </MetaText>
+        )}
+
+        {hasPanel && (
+          <MetaText direction="column">
+            Dashboard and panel
+            <Link href={makePanelLink(dashboardUID, panelID)} size="sm" external>
+              View panel
+            </Link>
+          </MetaText>
+        )}
+
+        {hasDashboardNoPanel && (
+          <MetaText direction="column">
+            Dashboard
+            <Link href={makeDashboardLink(dashboardUID)} size="sm" external>
+              View dashboard
+            </Link>
+          </MetaText>
+        )}
+
+        {interval && (
+          <MetaText direction="column">
+            Evaluation interval
+            <Text color="primary">Every {interval}</Text>
+          </MetaText>
+        )}
+
+        {/* TODO truncate, maybe build in to component? */}
+        {!isEmpty(labels) && (
+          <MetaText direction="column">
+            Labels
+            <AlertLabels labels={labels} size="sm" />
+          </MetaText>
+        )}
+      </Stack>
+    </>
+  );
+};
+
 interface BreadcrumbProps {
   folder: string;
   evaluationGroup: string;
 }
 
-const BreadCrumb = ({ folder, evaluationGroup }: BreadcrumbProps) => (
-  <Stack alignItems="center" gap={0.5}>
-    <Text color="secondary">
-      <Icon name="folder" />
-    </Text>
-    <Text variant="body" color="primary">
-      {folder}
-    </Text>
-    <Text variant="body" color="secondary">
-      <Icon name="angle-right" />
-    </Text>
-    <Text variant="body" color="primary">
-      {evaluationGroup}
-    </Text>
-  </Stack>
-);
+const createListFilterLink = (values: Array<[string, string]>) => {
+  const params = new URLSearchParams([['search', values.map(([key, value]) => `${key}:"${value}"`).join(' ')]]);
+  return createUrl(`/alerting/list?` + params.toString());
+};
+
+const BreadCrumbs = ({ folder, evaluationGroup }: BreadcrumbProps) => {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <Stack alignItems="baseline" gap={0}>
+      <Text color="secondary">
+        <Icon name="folder" />
+      </Text>
+      <LinkButton size="sm" variant="secondary" fill="text" href={createListFilterLink([['namespace', folder]])}>
+        {folder}
+      </LinkButton>
```

Comment:
you said the breadcrumbs you added did not link to any pages but this looks like a link to me :)

Reply:
if you need any help with how to add dynamic breadcrumb "parents" to a page let me know, it's a bit complex/contrived as it has to be done via the "parentItem" on the pageNav

Reply:
Ah sorry I should have been clearer – what I meant is that the current paradigm seems to link each breadcrumb item to a separate physical "page" and hadn't found any exceptions to that. I don't feel strongly attached to the separate breadcrumb and if we can incorporate it in to the existing breadcrumbs that'd be great! 👍